app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -631,15 +634,12 @@ class StudentForm extends React.Component {
                 {this.displayErrorMessage("gender")}
               </FormGroup>
 
-              <FormGroup validationState={this.getValidationState("birthday")}>
-                <ControlLabel>Birthday</ControlLabel>
-                <Datetime
-                  dateFormat="MM/DD/YYYY"
-                  timeFormat={false}
-                  inputProps={{placeholder: "MM/DD/YYYY"}}
-                  onChange={(moment) => this.handleDatetimeChange(moment, 'birthday')}/>
-                {this.displayErrorMessage("birthday")}
-              </FormGroup>
+              <FormatInput 
+                formName             = "Birthday"
+                inputId              = "birthday"
+                handleDatetimeChange = { (moment) => this.handleDatetimeChange(moment, "birthday") }
```

Comment:
Ok 👍 , so we're not having the datepicker anymore right? Sounds good! I think Forte thought it was weird to use too